ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPool.java

```diff
@@ -870,12 +875,19 @@ private void executeLoadFromDisk() {
                     .takeWhile(unused -> !isCancelled.get())
                     .map(
                         line -> {
-                          final boolean isLocal = line.charAt(0) == 'l';
+                          int i = 0;
+                          final var sbScore = new StringBuilder();
+                          while ("1234567890-".indexOf(line.charAt(i)) >= 0) {
+                            sbScore.append(line.charAt(i++));
+                          }
+                          final byte score =
+                              sbScore.isEmpty() ? MAX_SCORE : Byte.parseByte(sbScore.toString());
```

Comment:
maybe would be clearer to extract 878-884 to a method?

Reply:
done